runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -311,78 +302,4 @@ private static void applyLatencyTrackingInterval(
     long latencyTrackingInterval = options.getLatencyTrackingInterval();
     config.setLatencyTrackingInterval(latencyTrackingInterval);
   }
-
-  /**
-   * Remote stream environment that supports job execution with restore from savepoint.
-   *
-   * <p>This class can be removed once Flink provides this functionality.
-   *
-   * <p>TODO: https://issues.apache.org/jira/browse/BEAM-5396
-   */
-  private static class BeamFlinkRemoteStreamEnvironment extends RemoteStreamEnvironment {
```

Comment:
This cannot be removed yet. The feature is only present in 1.8.

Reply:
BeamFlinkRemoteStreamEnvironment has not been removed. Actually it has been moved to runner 1.7. The reason is that in 1.10 we don't need it any more. What do you think?

Reply:
I didn't see that. In this case 👍